client/v2/algod/dryrun.go

```diff
@@ -0,0 +1,24 @@
+package algod
+
+import (
+	"context"
+
+	"github.com/algorand/go-algorand-sdk/client/v2/common"
+	"github.com/algorand/go-algorand-sdk/client/v2/common/models"
+)
+
+// TealDryRun /v2/teal/dryrun
+type TealDryRun struct {
+	c      *Client
+	rawobj []byte
```

Comment:
In the spec, there is a second parameter that can go to the body. Why is it missing here?
            "description": "Transaction (or group) and any accompanying state-simulation data",                                                              
            "name": "jsonobj",                                                                                                                               
            "in": "body",                                                                                                                                    
            "schema": {                                                                                                                                      
              "$ref": "#/definitions/DryrunRequest"                                                                                                          
            }

Reply:
correct. this is a way to express in OAS2 the fact the endpoint accepts either blob or json object. I'm open to suggestions how the spec can be changed.

Reply:
Fixed the spec, now it has only one in-body parameter